packages/devtools_app/lib/src/primitives/enum_utils.dart

```diff
@@ -27,7 +27,8 @@ class EnumUtils<T> {
   final Map<String, T> _lookupTable = {};
   final Map<T, String> _reverseLookupTable = {};
 
-  T? enumEntry(String enumName) => _lookupTable[enumName];
```

Comment:
All errors in this PR are runtime errors, that were not caught by tests. Would it be right to cover the fixes with tests? May be in a separate CL, to speed up the fix landing.

Reply:
added some tests and filed https://github.com/flutter/devtools/issues/3991 to track the meta issue of improving devtools testing